Zend/zend_builtin_functions.stub.php

```diff
@@ -117,11 +117,15 @@ function set_error_handler(?callable $callback, int $error_levels = E_ALL) {}
 
 function restore_error_handler(): true {}
 
+function get_error_handler(): mixed {}
```

Comment:
Can't you have a return type of `?callable` ?

Reply:
Indeed, that's better